rmf_robot_sim_gz_plugins/src/TeleportIngestor.cpp

```diff
@@ -237,9 +443,9 @@ void TeleportIngestorPlugin::init_non_static_models_poses(
     const components::Static* is_static
     ) -> bool
     {
-      if (!is_static->Data() && name->Data() != _ingestor_common->_guid)
+      if (!is_static->Data() && name->Data() != _guid)
       {
-        _ingestor_common->non_static_models_init_poses[name->Data()] =
+        _non_static_models_init_poses[name->Data()] =
         convert_pose(
           pose->Data());
```

Comment:
We should just be able to use `gz::math` types rather than have to convert to `Eigen`. This is left from when we wanted to implement the bulk of the logic in a "non gz library", hence used `Eigen`, but is not necessary anymore and just adds complexity.